ADVANCED.md

```diff
@@ -723,40 +804,17 @@ return <PagingInfo view={PagingInfoView} />;
 
 **It will be helpful to read the [Headless Core](#headless-core) section first.**
 
-All Components support two hooks for customizing their behavior.
-
-1. `mapContextToProps` - Override the Context before it is passed to your Component as
-   props.
-2. `mapViewProps` - Lets you overrides view props before they are passed to the view.
+We have two primary recommendations for customizing component behavior:
 
-These allow you to override, modify, or even add completely new props.
+1. Overriding state and action props before they are passed to your Component, using `mapContextToProps`.
+2. Overriding props before they are passed to your Component's view.
 
-- These follow the same patterns as `mapStateToProps` in [Redux](https://redux.js.org/).
-- These **MUST** be immutable functions, if you directly update the props or context, you will have major issues in your application.
+### mapContextToProps
 
-To visualize these hooks:
+Every component supports a `mapContextToProps` prop, which allows you to modify state and actions
```

Comment:
There are a few cases where casing is inconsistent: 

1. component/Component
2. (Autocomplete) results/Results
3. (Autcomplete) suggestions/Suggestions

Reply:
Cleaned this up. In the rest of the doc we use "Component" so I made that consistent. I used lowers case for autocomplete results and suggestions.